### DISCUSSION TITLE
Hi thank you. You did awesome work and it works perfect

### DISCUSSION THREAD

**Discussion by suryajenson** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#18] [Q&A]
I was wondering can we able to change this mona Lisa icon?

**Comment by HerMajestyDrMona** [comment]
@suryajenson Yes. You would need to compile the program yourself, but overwrite the `Windows11DragAndDropToTaskbarFix.ico` file with some different icon.

I have permission to use this graphic from the autor: Martin Morano. In my opinion it fits perfectly.
We can not use any Windows logo as the program icon, as it's copyrighted: https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks
> Don’t use Microsoft’s logos, icons, or designs, in any manner.

... And I was out of ideas what it could be. After all it's supposed to be a simple fix, not some advanced tweaker program.

If you don't like the icon, maybe you don't need it at all? You could configure the program not to display the Tray Icon (as in previous versions of the fix): [CONFIGURE](https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/blob/main/CONFIGURATION.md#should-the-program-show-the-tray-icon-in-the-system-tray)

**Reply by suryajenson** [reply]
Thanks for explaining same doubt for my why they not updated this feature its not that hard as a designer it so annoying drag and drop not there thanks to you it work perfectly keep up the good work

**Reply by HerMajestyDrMona** [reply]
No problem. Enjoy the program :)

**Reply by HerMajestyDrMona** [reply]
@suryajenson The alternative program icon is available since [ver. 1.9](https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/blob/main/CHANGELOG.md#ver-1900----2021-11-15-0600-cet)

**Comment by hazymat** [comment]
Thanks for making this. I am genuinely baffled why in 2021 I cannot drag and drop in my operating system, when it's been possible for ... decades before now.

**Reply by HerMajestyDrMona** [reply]
I'm happy that you like the program. Thank you for the positive feedback.
